jfrog-cli/jfrog/buildtools_test.go

```diff
@@ -604,6 +604,59 @@ func TestNpm(t *testing.T) {
 	inttestutils.DeleteBuild(artifactoryDetails.Url, tests.NpmBuildName, artHttpDetails)
 }
 
+func TestGetJcenterRemoteDetails(t *testing.T) {
+	createServerConfigAndReturnPassphrase()
+
+	unsetEnvVars := func() {
+		os.Unsetenv(utils.JCenterRemoteServerEnv)
+		os.Unsetenv(utils.JCenterRemoteRepoEnv)
+	}
+	unsetEnvVars()
+	defer unsetEnvVars()
+
+	// The utils.JCenterRemoteServerEnv env var is not set, so extractor1.jar should be downloaded from jcenter.
+	downloadPath := "org/jfrog/buildinfo/build-info-extractor/extractor1.jar"
+	expectedRemotePath := path.Join("bintray/jcenter", downloadPath)
+	artDetails, remotePath, err := utils.GetJcenterRemoteDetails(downloadPath)
```

Comment:
This line repeat 4 times. You can use it inside validateJcenterRemoteDetails().

Reply:
Nice catch! Done.